contrib/IECoreMantra/src/IECoreMantra/procedural/Procedural.cpp

```diff
@@ -65,32 +65,6 @@ static void initialisePython()
 	}
 	Py_Initialize();
 	PyEval_InitThreads();
-		try
```

Comment:
This code only get executed if python has not be initialized yet. The problem was that if python was initialized somewhere else, then the module g_mainModule would be a pyobject None, which would then throw a keyerror for "IECore" later on. So using g_mainModule or g_mainModuleNamespace as it is wasn't really working. If we use g_mainModuleNamespace then we need some other check besides checking if python has been initialized or not. But then at that point maybe it would be better to have a static pyobject g_ieCoreModule or something, and then check if that's None in this pyInitialize, and then if not, import IECore.